spec.html

```diff
@@ -42183,95 +42301,6 @@ <h1>unescape ( _string_ )</h1>
       </emu-annex>
     </emu-annex>
 
-    <emu-annex id="sec-additional-properties-of-the-object.prototype-object">
```

Comment:
This PR undefines this id. Would it make sense as an oldid of sec-properties-of-the-object-prototype-object ?